config/routes.rb

```diff
@@ -554,6 +554,8 @@
   constraints(id: /[^\/]+/) do
     match 'new/hosts/:id' => 'react#index', :via => :get, :as => :host_details_page
   end
+  match 'new/hosts/' => 'react#index', :via => :get
```

Comment:
This should be `/hosts/new` to match the other new host routes (`/hosts/new/:id`)

Reply:
good point.. Ok, leave as-is then :smile: